spec/factories/comment_factory.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :comment do
+    sequence(:body) { |n| " Some great comment that user#{n} left"
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.